src/main/scala/firrtl/annotations/Annotation.scala

```diff
@@ -127,130 +127,7 @@ trait MultiTargetAnnotation extends Annotation {
-      errorIllegalAnno("EmittedVerilogCircuitAnnotation")
-    case LegacyAnnotation(_,_,value) if value.startsWith("emittedVerilogModule") =>
-      errorIllegalAnno("EmittedVerilogModuleAnnotation")
-    // People shouldn't be trying to pass deleted annotations to Firrtl
-    case LegacyAnnotation(_,_,OldDeletedRegex(_,_)) => errorIllegalAnno("DeletedAnnotation")
-    // Some annotations we'll try to support
-    case LegacyAnnotation(named, t, _) if t == classOf[InlineInstances] => InlineAnnotation(named)
-    case LegacyAnnotation(n: ModuleName, t, outputConfig) if t == classOf[ClockListTransform] =>
-      ClockListAnnotation(n, outputConfig)
-    case LegacyAnnotation(CircuitName(_), transform, "") if transform == classOf[InferReadWrite] =>
-      InferReadWriteAnnotation
-    case LegacyAnnotation(_,_,PinsRegex(pins)) => PinAnnotation(pins.split(" "))
-    case LegacyAnnotation(_, t, value) if t == classOf[ReplSeqMem] =>
-      val args = value.split(" ")
-      require(args.size == 2, "Something went wrong, stop using legacy ReplSeqMemAnnotation")
-      ReplSeqMemAnnotation(args(0), args(1))
-    case LegacyAnnotation(c: ComponentName, transform, "nodedupmem!")
-      if transform == classOf[ResolveMemoryReference] => NoDedupMemAnnotation(c)
-    case LegacyAnnotation(m: ModuleName, transform, "nodedup!")
-      if transform == classOf[DedupModules] => NoDedupAnnotation(m)
-    case LegacyAnnotation(c: ComponentName, _, SourceRegex(pin)) => SourceAnnotation(c, pin)
-    case LegacyAnnotation(n, _, SinkRegex(pin)) => SinkAnnotation(n, pin)
-    case LegacyAnnotation(m: ModuleName, t, text) if t == classOf[BlackBoxSourceHelper] =>
-      val nArgs = 3
-      text.split("\n", nArgs).toList match {
-        case "resource" :: id ::  _ => BlackBoxResourceAnno(m, id)
-        case "inline" :: name :: text :: _ => BlackBoxInlineAnno(m, name, text)
-        case "targetDir" :: targetDir :: _ => BlackBoxTargetDirAnno(targetDir)
-        case _ => errorIllegalAnno("BlackBoxSourceAnnotation")
-      }
-    case LegacyAnnotation(_, transform, "noDCE!") if transform == classOf[DeadCodeElimination] =>
-      NoDCEAnnotation
-    case LegacyAnnotation(c: ComponentName, _, "DONTtouch!") => DontTouchAnnotation(c.toTarget)
-    case LegacyAnnotation(c: ModuleName, _, "optimizableExtModule!") =>
-      OptimizableExtModuleAnnotation(c)
-    case other => other
-  }
-  def convertLegacyAnnos(annos: AnnotationSeq): AnnotationSeq = {
-    var warned: Boolean = false
-    annos.map {
-      case legacy: LegacyAnnotation =>
-        val annox = convertLegacyAnno(legacy)
-        if (!warned && (annox ne legacy)) {
-          val msg = s"A LegacyAnnotation was automatically converted.\n" + (" "*9) +
-            "This functionality will soon be removed. Please migrate to new annotations."
-          StageUtils.dramaticWarning(msg)
-          warned = true
-        }
-        annox
-      case other => other
-    }
-  }
-}
+object Annotation
```

Comment:
How would we go about removing this object?

Reply:
I don't know why anyone would be relying on it, but we'd have to deprecate. I'm not sure that we should though, it's a reasonable place to put utilities or implicit conversions if needed.

Reply:
Sounds good. Mostly just a curiosity... Yes, this could be reuse for utilities later.